src/js/models/layer.js

```diff
@@ -343,8 +357,7 @@ define(function (require, exports, module) {
             mode: "passThrough",
             proportionalScaling: false,
             isArtboard: false,
-            isLinked: false,
-            hasLayerEffect: false
```

Comment:
Groups can have effects as well.. so I can reproduce some of the play errors you were getting by adding/removing effects from group layers.